src/control/cmd/dmg/utils_test.go

```diff
@@ -24,11 +24,12 @@
 package main
 
 import (
+	"fmt"
 	"testing"
 
 	. "github.com/daos-stack/daos/src/control/client"
-	. "github.com/daos-stack/daos/src/control/common"
 	ctlpb "github.com/daos-stack/daos/src/control/common/proto/ctl"
+	"github.com/google/go-cmp/cmp"
```

Comment:
misplaced import

Reply:
fixed in https://github.com/daos-stack/daos/pull/1357